src/main/java/com/tradeshift/service/HelloWorldServiceImpl.java

```diff
@@ -0,0 +1,61 @@
+package com.tradeshift.service;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import com.tradeshift.model.Content;
+import com.tradeshift.model.DBMessage;
+import com.tradeshift.model.Message;
+import com.tradeshift.model.RecentModel;
+
+public class HelloWorldServiceImpl implements HelloWorldService {
+
+    @Autowired
+    private MessagesDAO messagesDAO;
+
+    public HelloWorldServiceImpl() {
+    }
+
+    public HelloWorldServiceImpl(MessagesDAO messagesDAO) {
+        this.messagesDAO = messagesDAO;
+    }
+
+    @Override
+    public String getHelloWorldMessage(String name){
+
+        return "hello " + name;
+    }
+
+    @Override
+    public void insert(String name) {
+        if(name == null && name.isEmpty()) {
+            throw new NullPointerException("String is null or empty");
```

Comment:
Do not throw null pointers where the value isn't null. Especially here you should (if anything) throw an `IllegalArgumentException` or other instead.